checker/jtreg/nullness/issue824/Class2.java

```diff
@@ -1,9 +1,10 @@
 /*
  * @test
  * @summary Test case for Issue 824 https://github.com/typetools/checker-framework/issues/824
+ * The defaults for type variable upper bounds with type Object changed since the issue was filed.  So, this test case has been changed so that annotations on type variable bounds in stub files is still tested.
```

Comment:
As far as I can tell, you have to submit a new review.  (See https://help.github.com/articles/approving-a-pull-request-with-required-reviews/)